demos/src/demo.scss

```diff
@@ -0,0 +1,21 @@
+$o-typography-is-silent: false;
+@import 'o-typography/main';
+
+$o-topper-is-silent: false;
+@import '../../main';
+
+@include oTopperThemes;
+
+body {
+	@include oColorsFor(page, background);
+}
+
+.o-topper {
+	// TODO: add to next-article, discuss whether o-topper should do it itself
+	-webkit-font-smoothing: antialiased; // sass-lint:disable-line no-vendor-prefixes
+}
+
+.tags--large {
+	// TODO: should this be part of o-topper? should it be done by the opinion theme?
+	@include oTypographySize($scale: 2, $line-height: 28px);
```

Comment:
```.o-topper__tags--large,
.o-topper__topic--large {
    @include oTypographySize($scale: 2, $line-height: 28px);
}```
?

Reply:
i'm not sure it can be done theme level. Next's logic for which tags to make large is complex ([lines 13, 23 and 34](https://github.com/Financial-Times/next-article/blob/ca348a410df507513ed420fb0aa0b43221a9eb7f/views/partials/topper/components/tags.html#L13)).

Reply:
...hit submit too early. there's a whole set of assumptions there that feel difficult to dissect. i'll add it as a utility class (maybe just a single `.o-topper__tag--large`? since internally these all extend `%_o-topper__tag`. is it weird to have a modifier for an element that doesn't exist? that is used on multiple elements?)

Reply:
actually you know what i'm gonna do it properly with the themes